src/components/manage/Blocks/Listing/FullCalendar.jsx

```diff
@@ -59,9 +60,13 @@ const FullCalendarListing = ({ items, moment: momentlib, ...props }) => {
 
   let recurrences = [];
 
+  const contentConverters =
+    config.blocks.blocksConfig.fullcalendar.contentConverters;
+
   let events = items
     .filter((i) => {
-      if (i['@type'] !== 'Event') return false;
+      if (!Object.keys(contentConverters).includes(i['@type'])) return false;
+
```

Comment:
Why not filtering by event like types, which are items with 'start' and 'end', then skipping the introduction of a 'contentConverters' function?

Reply:
The `contentConverters` provides a flexible way to let the developer decide which kind of items he wants to show in the calendar.

For instance, I have items that are not events, that don't have 'start' and 'end' attributes. That's why I built this function, as if it was a ZCA adapter, this way you can decide which fields you want to use as `title, `start` and `end` and this way show the item on the calendar.

Reply:
IMHO it's the job of the backend to provide common practice attributes like 'start' and 'end' according IEventBasics behavior.
By now you are asking the integrator to provide a configuration for any other content type than 'event'. There should be a fallback for IEventBasic behavior enhanced content types. They do have a 'start' attribute.

Reply:
Well, right now the existing implementation only supports Event content-types (like in the deleted condition). I have kept the implementation for Events, and extend it for any other content-type.

Reply:
Your solution lacks a default / fallback for instances that already provide 'start', for example  IEventBasics enhanced content types.

Again: I would prefer to only loosen the filter condition  from `if (i['@type'] !== 'Event')` to just `if (!i.start)`. No extra configuration. But instead on backend providing custom practice attributes of the IEventBasics.

Reply:
ah Ok. I wasn't understanding your request. That's doable of course.